### PR TITLE
Gas the meatbags race war now.

### PR DESCRIPTION
**Pull request by PJB3005**
- Removes vox healing from nitrogen reagent.
- Made vox equivalents for Dexalin (Vexalin), Dexalin plus (Vexalin Plus), and Inaprovaline (Invaxovaline).
  - They're made the same as their meatbag counterpart except with nitrogen instead of oxygen.
  - Only work for vox, meatbags completely unaffected.
- Dex (plus) now kills vox properly.
- Adds a _truly majestic_ shriek to vox, from paradise.

DNM I need a changelog, fix that issue @Midaychi pointed, TEST this, and get the wiki update ready. (when pomf wants to merge it.)

### CODE REVIEW

**Comment by Midaychi** [comment]
*scream

**Comment by PJB3005** [comment]
*shriek

**Comment by Midaychi** [comment]
the shriek happens when they get hurt-screamed right?

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -1771,17 +1799,57 @@
...
+
+	M.adjustOxyLoss(-2 * REM)
+	if(holder.has_reagent("lexorin"))
+		holder.remove_reagent("lexorin", 2 * REM)
+	..()
+
+/datum/reagent/vexalinp
+	name = "Vexalin Plus"
+	id = "vexalinp"
+	description = "Dexalin Plus is used in the treatment of nitrogen deprivation. Its highly effective."
```
Comment (Midaychi) [inline code/modules/reagents/Chemistry-Reagents.dm]: Typo? 
	description = "Vexalin Plus is used in the treatment of nitrogen deprivation. Its highly effective."

Comment (PJB3005) [inline code/modules/reagents/Chemistry-Reagents.dm]: whoops.

**Comment by PJB3005** [comment]
When doing *scream it happens, I think hurt scream just calls *scream.

**Comment by SkowronX** [comment]
I suggest differentiating Inaprov and Invaprov a little more.

**Comment by Probe1** [comment]
what Skow said

**Comment by Probe1** [comment]
also @PJB3005 did you change the iron replenishing their blood instead of copper?  pls include.

**Comment by Iamgoofball** [comment]
why didn't you just make them subtypes of the original chems

woulda saved you some copypasta and you can just OOP out the parts you want changed

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -1627,6 +1649,10 @@
 	if(!M) M = holder.my_atom
 	if(holder.has_reagent("inaprovaline"))
 		holder.remove_reagent("inaprovaline", 2*REM)
+
+	if(holder.has_reagent("invaprovaline"))
+		holder.remove_reagent("invaprovaline", 2*REM)
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents.dm]: reasons why chemistry code is shit #23403478279834

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 3]
```diff
@@ -655,6 +655,28 @@
 	holder.remove_reagent(src.id, 0.5 * REAGENTS_METABOLISM)
 	return
 
+//Not to be confused with inaprovaline
+/datum/reagent/invaprovaline
+	name = "Invaprovaline"
+	id = "invaprovaline"
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents.dm]: You need to change this ID. Make it "inaprov_vox" or something.

This is such a minor change people are literally going to confuse it with a typo.

**Comment by Iamgoofball** [comment]
when a race has 3 dedicated copypasta versions of a chem just for that race alone because the other chems kill it

you're so deep in the snowflake not OOP code that any semblance of coding quality or standards is dead

**Comment by FudgePucker** [comment]
Wouldn't dex+ outright kill a vox the second it entered the blood stream?
It removes O2 deprivation instantly, so you would assume that it is some magical super drug that instantly puts O2 into all of your cells magically, so to a vox it would cause every single cell in their body to commit sudoku.
...
...
Yes I do want to be able to instakill snowflakes.

**Comment by SkowronX** [comment]
Almost forgot, add support for all non oxygen breathers.

**Comment by Zuhayr** [comment]
I was excited that maybe Paradise had made a better shriek sound but it's just the Bay one. :(

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents.dm]: Nice copy pasta.

**Comment by PJB3005** [comment]
@FudgePucker that would mean people would be able to insta kill vox with a syringe gun.

**Comment by Midaychi** [comment]
With this change, you should be able to knock a vox into crit by feeding them a pill wih 50 units of dex plus

**Comment by FudgePucker** [comment]
@PJB3005 
>That would mean people would be able to insta kill vox with a syringe gun

WEW LAD
@Midaychi 
Sticking 50 units into anybody (Exept plantbros) would kill them eventually, its just you would have to wait for the dex+ to run out before they go into crit from toxin damage.

**Comment by ComicIronic** [comment]
Vinaprovaline?

**Comment by ComicIronic** [comment]
Goof is also right about the copypasta, it should really be using inheritance.

**Pull request closed without merging**